modxcms/revolution
# Fix missing labels and descriptions in TV quick create/edit window (revised)

### What does it do?
(Note: This is a revised partial of PR #15950.)

Addresses referenced issue by moving select TV entries to the default topic to make them available application-wide.

### Why is it needed?

There are missing labels and descriptions in the quick create/edit (QCE) form when working with a TV (when the main view is anything but the TV panel, as that's the only place where the TV-specific lexicons are loaded).

### How to Test
1. From your terminal app, run `grunt build` from within the `_build/templates/default` folder and clear your browser cache. (The build step is only needed if you are running the manager with compress_js on.)
2. Create at least one TV using the contextual quick create feature. Then, use quick edit on that TV (or others you may have created in your test environment). Try the QCE feature from different pages within the manager to verify labelling and descriptions remain visible.

### Special Note
Ultimately, more work will need to be done in a separate PR to bring the tv type-specific field display of the main TV panel to the associated QCE window (for the _default value_ and _input options_ fields).

### Related issue(s)/PR(s)
Resolves #15947
